src/core/blueprint/operators/http.rs

```diff
@@ -94,7 +94,20 @@ pub fn compile_http(
                     is_list,
                     dedupe,
                 })
+            };
+
+            if let Some(select) = &http.select {
+                let dynamic_value = match DynamicValue::try_from(select) {
+                    Ok(dynamic_value) => dynamic_value.prepend("args"),
+                    Err(_) => {
+                        return Valid::fail(format!("syntax error when parsing `{:?}`", select))
+                    }
+                };
+
+                io = io.pipe(IR::Dynamic(dynamic_value));
             }
```

Comment:
some duplicate code here in http.rs and gRPC. How about we create a new file `select.rs` and move this logic there.